apps/frontend/src/renderer/components/Sidebar.tsx

```
@@ -271,6 +274,33 @@ export function Sidebar({
+        <Tooltip key={item.id}>
+          <TooltipTrigger asChild>
+            <button
+              onClick={() => handleNavClick(item.id)}
+              disabled={!selectedProjectId}
+              className={cn(
+                'flex h-11 w-16 items-center justify-center rounded-lg text-sm transition-all duration-200',
+                'hover:bg-accent hover:text-accent-foreground',
+                'disabled:pointer-events-none disabled:opacity-50',
+                isActive && 'bg-accent text-accent-foreground'
+              )}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)